pkg/sql/logictest/testdata/logic_test/mixed_version_upgrade_preserve_ttl

```diff
@@ -1,14 +1,24 @@
 # LogicTest: cockroach-go-testserver-upgrade-to-master
 
+# This test verifies that when a cluster is upgraded, it preserves the TTL
+# job configuration on a table.
+
+query T
+SELECT version FROM [SHOW CLUSTER SETTING version]
+----
+23.1
```

Comment:
ah so we have to `--rewrite` this everytime the next major version changes (and change the regex below)?

is there another way of asserting this that doesn't have to be maintained :o (genuinely curious)

Reply:
> is there another way of asserting this that doesn't have to be maintained :o (genuinely curious)

i couldn't think of one, but this manual step is something that should only happen twice per year, and thanks to the first assertion i added here, now it will fail loudly the next time someone bumps the major version.

Reply:
twice a year, i will look forward to this